src/panels/lovelace/editor/config-elements/hui-gauge-card-editor.ts

```diff
@@ -86,6 +87,26 @@ export class HuiGaugeCardEditor
             },
           },
         },
+        {
+          name: "attribute",
+          selector: {
+            attribute: {
+              entity_id: entityId,
+              hide_attributes: [
+                // exclude some of base entity attributes
+                "friendly_name",
+                "unit_of_measurement",
+                "icon",
+                "entity_picture",
+                "hidden",
+                "assumed_state",
+                "device_class",
+                "state_class",
+                "restored",
+              ],
```

Comment:
I think it makes sense to move the list from `src/panels/config/automation/condition/types/ha-automation-condition-numeric_state.ts` to `src/data/entity_attributes.ts` as a `const` and use it here as well

Reply:
done [b1826e8](https://github.com/home-assistant/frontend/pull/25258/commits/b1826e82637bebb05a720df0a539f56133a433c8). Checked manually that numeric condition in automation - works as before